app/controllers/admin/merchants_controller.rb

```diff
@@ -6,4 +6,22 @@ def index
     def show
         @merchant = Merchant.find(params[:id])
     end
-end
\ No newline at end of file
+
+    def edit
+        @merchant = Merchant.find(params[:id])
+    end
+
+    def update
+        merchant = Merchant.find(params[:id])
+        if merchant.update(merchant_params)
+            flash[:notice] = "Merchant information is updated successfully"
+            redirect_to admin_merchant_path(merchant)
+        end
+    end
+
+    private
+
+    def merchant_params
+        params.require(:merchant).permit(:name)
```

Comment:
What does the require method do in this section? I know that params.permit(params[:name]) would just as well correct?